### PR TITLE
Update `ko`

### PR DESCRIPTION
**Pull request by Terrorboy**
Edit the changed text

### CODE REVIEW

File: Language/ko/HTTP.php [hunk 1]
```diff
@@ -15,7 +15,7 @@
     'missingCurl'     => 'CURLRequest 클래스를 사용하기 위해서 CURL이 반드시 활성화되어야 합니다.', // 'CURL must be enabled to use the CURLRequest class.'
     'invalidSSLKey'   => 'SSL Key를 설정할 수 없습니다. {0} 경우는 유효하지 않는 파일입니다.', // 'Cannot set SSL Key. {0} is not a valid file.'
     'sslCertNotFound' => '해당 파일에서 SSL 인증서를 찾을 수 없습니다: {0}', // 'SSL certificate not found at: {0}'
-    'curlError'       => '{0} : {1}',
+    'curlError'       => '{0} : {1}', // {0} : {1}
```
Comment (paulbalandan) [inline Language/ko/HTTP.php:18]: Why do you need this comment if it is the same?

Comment (Terrorboy) [reply Language/ko/HTTP.php:18]: https://github.com/codeigniter-kr/ci4lang Added inline comments to use the program.

Comment (Terrorboy) [reply Language/ko/HTTP.php:18]: image

Comment (paulbalandan) [reply Language/ko/HTTP.php:18]: Nice script you got there! Thanks.

**Review by paulbalandan — APPROVED** [review]

**Pull request merged**